component/kyverno.jsonnet

```
@@ -15,6 +16,23 @@ local services = std.parseJson(kap.yaml_load_stream(manifests_path + '/service.y
 local deployment = std.parseJson(kap.yaml_load(manifests_path + '/deployment.yaml'));
 local configmap = std.parseJson(kap.yaml_load(manifests_path + '/configmap.yaml'));
 
+
+local nodeSelectorConfig(role) =
+  if role == null then
+    {}
+  else
+    local label = 'node-role.kubernetes.io/%s' % role;
+    {
+      nodeSelector+: {
+        [label]: '',
+      },
+      tolerations+: [ {
+        key: label,
+        operator: 'Exists',
+        effect: 'NoSchedule',
+      } ],
+    };
+
 local objects = [] +
                 roles +
```

Having the empty array seems redundant (mostly unrelated to this PR):

```suggestion
local objects =
  roles +
```

Please always consider inserting a line break, if the Jsonnet formatter wastes this much horizontal space.